backend/docker_ops.py

```
@@ -1,383 +1,642 @@
-
-            logger.info(f"Updated compose file paths in {compose_file_path}")
+            client = self.get_docker_client(target_host, ssh_user)
+            success_count = 0
+            
+            for container_info in container_infos:
+                try:
+                    # Build container configuration
+                    container_config = self._build_container_config(container_info, volume_mapping)
+                    
+                    # Create and start container
+                    container = client.containers.run(**container_config)
```

Ruff (0.11.9)

Local variable `container` is assigned to but never used

Remove assignment to unused variable `container`

(F841)